sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/block/degrade/DegradeRule.java

```diff
@@ -104,11 +103,11 @@ public DegradeRule setCount(double count) {
         return this;
     }
 
-    public boolean isCut() {
+    public AtomicBoolean getCut() {
```

Comment:
Here it's better to be compatible with the original method. You can wrap AtomicBoolean with the original method.

```suggestion
    public boolean isCut() {
        return cut.get();
    }
```

Also the getter/setter method may not necessary to be `public`. For test we can just make it package-private.